lenses/devfsrules.aug

```diff
@@ -0,0 +1,24 @@
+module DevfsRules =
+
+  autoload xfm
+
+  let comment  = IniFile.comment IniFile.comment_re "#"
+
+  let eol = Util.eol
+
+  let line_re = /[^][#; \t\n][^#;\n]*[^#; \t\n]/
+  let entry = [ seq "entry" . store line_re . (eol | comment) ]
```

Comment:
can you explain this one?

can we add a negative test?

Reply:
Lines don't really have a key/value pattern, it's more entire lines to store, so I'm using a `seq` approach for that, with a counter called "entry", which is reset in the `title` definition so that entries are numbered inside each section. We then store `line_re`, which is anything but characters that will clash with `eol` or `comment`.